test/server/cards/15.1-TL/ProceduralInterferece.spec.js

```diff
@@ -0,0 +1,93 @@
+describe('Procedural Interference', function() {
+    integration(function() {
+        beforeEach(function() {
+            this.setupTest({
+                phase: 'dynasty',
+                player1: {
+                    dynastyDiscard: ['aranat', 'daidoji-uji', 'procedural-interference', 'procedural-interference'],
+                    honor: 10
+                },
+                player2: {
+                    dynastyDiscard: ['aranat', 'daidoji-uji', 'imperial-storehouse', 'heavy-ballista'],
+                    honor: 10,
+                    provinces:['shameful-display', 'silent-ones-monastery']
+                }
+            });
+            this.interference = this.player1.findCardByName('procedural-interference');
+            this.interference2 = this.player1.findCardByName('procedural-interference');
```

Comment:
This is finding the same card (this.interference === this.interference2).

Use `this.player1.filterCardsByName('procedural-interference')[0]` and `[1]` to get the specific cards.